extensions/youtube-downloader/src/utils.ts

```diff
@@ -1,6 +1,20 @@
 import { getPreferenceValues } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
 import { formatDuration, intervalToDuration } from "date-fns";
 import isUrlSuperb from "is-url-superb";
+import { GET_ACTIVE_APP_SCRIPT, GET_LINK_FROM_BROWSER_SCRIPT, SUPPORTED_BROWSERS } from "./browser.js";
+
+type ExtensionPreferences = {
+  downloadPath: string;
+  ytdlPath: string;
+  ffmpegPath: string;
+  ffprobePath: string;
+  forceIpv4: boolean;
+  autoLoadUrlFromClipboard: boolean;
+  autoLoadUrlFromSelectedText: boolean;
+  autoLoadUrlFromBrowserActiveTab: boolean;
+  homebrewPath: string;
+};
```

Comment:
Do you know the `ExtensionPreferences` is in the `raycast-env.d.ts`? You don't need to define it here.

Reply:
For some reason, I get errors and thought it wasn't working properly. I'll remove the type definition.

Reply:
That's because the `raycast-env.d.ts` file is ignored by `.gitignore`. You need to run `npm run dev` or `npm run build` to generate this file.

Reply:
Fixed